docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+[[add-fleet-server-kubernetes]]
+= Deploy Fleet Server on Kubernetes
+
+[NOTE]
+====
+For ECK users, if your {stack} is orchestrated by ECK, we recommend to orchestrate the {fleet-server} through the operator. That simplifies the process as all needed resources will be configured by ECK.
```

Comment:
I understand what you're going for, but the start of this sentence is repetitive. I would remove the "For ECK users" part.

Reply:
Sure! That part together with the `certain knowledge required` was a bit of a draft / placeholder, and I had forgotten to review and update it. I'll do it now.

Reply:
I have updated it, let me know your thoughs @bmorelli25 :)